.github/workflows/CI-orchestrator.yml

```diff
@@ -50,7 +55,7 @@ jobs:
 
   set-overall-result:
     runs-on: ubuntu-latest
-    needs: [build-job, test-job, coverage-job, lint-format-job, zombienet-test-job, audit-job, check-act]
+    needs: [build-job, test-job, try-runtime-job, coverage-job, lint-format-job, zombienet-test-job, audit-job, check-act]
```

Comment:
I was thinking to avoid increasing CI complexity (on top of everything we already have):

- We can move this job into the tag-orchestrator
- For the orchestrator we can make this job running only on main branch, and only after: merging OR schedule runs couple of times a week